iOSDevUK/Controllers/Sessions/ProgrammeTableViewController.swift

```diff
@@ -86,6 +98,30 @@ class ProgrammeTableViewController: UIViewController, UITableViewDelegate, UITab
         }
     }
     
+    /// Find the current or next session for the current set of sessions and scroll to it.
+    func scrollToCurrentSession(animated: Bool) {
+        guard let fetchedResultsController = fetchedResultsController else {
+            return
+        }
+        
+        /// Find the session in the already-fetched results
+        let nextOrCurrentSession = fetchedResultsController.fetchedObjects?.first { session in
```

Comment:
I just noticed that the `Session` object already has a way to calculate the current and the next Session. Maybe you can use `Session.nowSession(forDate:)` and `Session.nextSession(forDate:)` here instead of reimplementing it?

Reply:
I saw those too, but it seems like a silly idea to run 2 Core Data queries to find this?